### DISCUSSION TITLE
PCL2的自定义主页联网更新怎么部署

### DISCUSSION THREAD

**Discussion by xtrackc1437** [Meloong-Git/PCL#1997] [提问] [answered]
如题 
文件直接放网站上去没用啊...

**Comment by MoYuan-CN** [comment, marked as answer]
需要使用静态文件部署服务 [^1]

举个例子：
我有个主页部署在 Gitee 仓库中，地址为 [这里](https://gitee.com/JingHai-Lab/ccxaml/blob/master/Custom.xaml)
然后需要获取文件的源数据（RAW），对应地址 [这里](https://gitee.com/JingHai-Lab/ccxaml/raw/master/Custom.xaml)
之后把源数据地址贴在楼上所述的地方即可使用

[^1]:可使用Gitee或Github仓库部署，若自己有相应服务也是可以的
